client/plots/plot.ssgq.js

```diff
@@ -65,7 +65,10 @@ export async function plotSingleSampleGenomeQuantification(
 		const q2 = termdbConfig.queries.singleSampleGbtk?.[q.singleSampleGbtk]
 
 		// description
-		holder.append('div').text(q.description || queryKey)
+		holder
+			.append('div')
+			.attr('data-testid', 'methylation_array_sand_box')
```

Comment:
methylation array is just one type of data that could be shown in ssgq. i prefer to rename id to `sjpp_ssgq_sandbox`

Reply:
Changed.